packages/create-turbo/src/transforms/update-commands-in-readme.ts

```diff
@@ -0,0 +1,44 @@
+import path from "node:path";
+import fs from "node:fs/promises";
+import { TransformError } from "./errors";
+import type { TransformInput, TransformResult} from "./types";
+
+const meta = {
+  name: "update-commands-in-readme",
+};
+
+export async function transform(args: TransformInput): TransformResult {
+  const { prompts, example } = args;
+
+  const isOfficialStarter =
+    !example.repo ||
+    (example.repo.username === "vercel" && example.repo.name === "turbo");
+
+  if (!isOfficialStarter) {
+    return { result: "not-applicable", ...meta };
+  }
+
+  const selectedPackageManager = prompts.packageManager;
+  const readmeFilePath = path.join(prompts.root, "examples", "basic", "README.md");
```

Comment:
This should just be:

```suggestion
  const readmeFilePath = path.join(prompts.root, "README.md");
```

Reply:
Updated the path as per suggestion, thanks :)